sys/dev_vga.c

```diff
@@ -14,10 +14,54 @@ static int dev_vga_palette_write(vnode_t *v, uio_t *uio) {
   return vga_palette_write((vga_device_t *)v->v_data, uio);
 }
 
+#define RES_CTRL_BUFFER_SIZE 32
+
+static int dev_vga_resolution_write(vnode_t *v, uio_t *uio) {
+  vga_device_t *vga = (vga_device_t *)v->v_data;
+  uio->uio_offset = 0; /* This file does not support offsets. */
+  uint16_t xres, yres;
```

Comment:
`vga_{get,set}_resolution` use `uint16_t` because I wanted to ensure that there are at least 12 bits for dimensions. But as it's unlikely that we'll ever want to run mimiker on an architecture with word size < 12 bits, I can probably use the generic `unsigned` instead.